packages/cli/src/ceramic-daemon.ts

```diff
@@ -388,6 +392,20 @@ export class CeramicDaemon {
     res.json(response)
   }
 
+  /**
+   * Render the most recent version of a stream's contents
+   */
+  async rawData (req: Request, res: Response): Promise<void> {
+    const opts = parseQueryObject(req.query)
+    const stream = await this.ceramic.loadStream(req.params.streamid, opts)
+    const state = StreamUtils.serializeState(stream.state)
```

Comment:
Instead of this, I think it would make more sense to add a `raw_data` accessor to the [Stream type](https://github.com/ceramicnetwork/js-ceramic/blob/881d7f0f17de820290ba6b5b7f4b19e00d2eed6c/packages/common/src/stream.ts#L130).  The `raw_data` accessor can work very similarly to the [metadata accessor](https://github.com/ceramicnetwork/js-ceramic/blob/881d7f0f17de820290ba6b5b7f4b19e00d2eed6c/packages/common/src/stream.ts#L145), just returning the `content` field instead of the `metadata` field.

Reply:
Done.